compiler/one-cmds/CMakeLists.txt

```diff
@@ -36,7 +36,11 @@ set(ONE_COMMAND_FILES
 )
 
 # TODO find better way for per-platform files
-if(ONE_UBUNTU_CODENAME_JAMMY)
+if(CMAKE_HOST_SYSTEM_PROCESSOR STREQUAL "aarch64")
```

Comment:
Maybe to check Ubuntu version + CPU Arch ?
```
if arch is aarch65
  if ubuntu_jammy
     A
  else
     B
  endif
else if arch is x86-64
  if ubuntu_jammy
     A
  else
     B
  endif
else
  fail
endif
```

Reply:
@seanshpark , applied. For aarch64, I'm currently only considering support for bionic, so I'm not branching by ubuntu version. I will expand later if needed.